tests/beacon/utils/test_random.py

```diff
@@ -11,16 +11,24 @@
     ),
     [
         (
-            [i for i in range(100)],
-            b"\x8d\x8dd\xb9\x84\x05*Bw\xf5\x87\xdd\x04nj\xee\x82\xfc9\xb9\x19\x82'x!#p];\xa5\x8e\xc9",  # noqa: E501
-            (87, 32, 76, 41, 61, 9, 44, 78, 70, 42, 8, 6, 79, 37, 4, 67, 3, 66, 55, 0, 69, 98, 38, 47, 20, 31, 48, 97, 34, 33, 96, 63, 91, 15, 1, 77, 39, 64, 58, 53, 82, 49, 26, 29, 5, 65, 19, 86, 74, 23, 46, 14, 10, 51, 85, 89, 57, 73, 71, 52, 88, 30, 16, 81, 24, 22, 36, 45, 12, 2, 62, 93, 18, 72, 83, 95, 13, 21, 27, 28, 80, 43, 68, 59, 94, 60, 99, 56, 92, 54, 25, 75, 7, 90, 50, 35, 11, 84, 17, 40),  # noqa: E501
+            tuple(range(100)),
+            b'\x32' * 32,
+            (92, 18, 80, 86, 39, 56, 81, 74, 68, 64,
+             52, 82, 26, 25, 45, 2, 66, 38, 35, 15,
+             46, 9, 65, 55, 85, 51, 53, 95, 57, 88,
+             43, 70, 22, 5, 31, 63, 90, 61, 89, 24,
+             6, 94, 12, 29, 84, 72, 42, 73, 40, 48,
+             16, 77, 69, 4, 14, 75, 21, 17, 54, 37,
+             98, 3, 60, 36, 76, 1, 97, 0, 44, 19,
+             83, 28, 41, 47, 13, 91, 93, 7, 71, 58,
+             11, 20, 50, 30, 34, 49, 33, 59, 79, 62,
+             67, 96, 78, 8, 10, 99, 87, 27, 32, 23),
         ),
         (
-            [i for i in range(12)],
-            b"&P\x18'\x91\xf6\x8f\xc8;<\xb8\x8f\x1d\x92?\xad4\xb5\xb8\x0b\xca\x8a'\x8e\xae'\xcf7\xa3(\xb2\x8d",  # noqa: E501
-            (7, 3, 2, 5, 11, 9, 1, 0, 4, 6, 10, 8),
-
-        )
+            tuple(range(12)),
+            b'\x23' * 32,
+            (11, 4, 9, 5, 7, 10, 2, 8, 0, 6, 3, 1),
```

Comment:
This test was from lighthouse's codebase. Maybe open an issue to try out https://github.com/status-im/eth2-testgen/blob/master/test_vectors/test_vector_shuffling.yml.

Reply:
issue opened here https://github.com/ethereum/py-evm/issues/1615